static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -121,20 +120,12 @@ const getRequestDataFragment = ({
 
 export default function IssueAlertOptions({
```

Comment:
This component is used within a modal that no longer appears to be in use. In any case, I tested the modal locally, and everything is working as expected.